release-resources/release-strategy.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseStrategy
+metadata:
+  name: e2e-fbc-strategy
+  namespace: 
+spec:
+  bundle: 'quay.io/hacbs-release/pipeline-fbc-release:main'
+  params:
+    - name: fromIndex
+      value: 'quay.io/scoheb/fbc-index-testing:latest'
+    - name: targetIndex
+      value: 'quay.io/scoheb/fbc-target-index-testing:latest'
+    - name: binaryImage
+      value: 'registry.redhat.io/openshift4/ose-operator-registry:v4.12'
+    - name: requestUpdateTimeout
+      value: '420'
+    - name: buildTimeoutSeconds
+      value: '480'
+  pipeline: fbc-release
+  policy: fbc-policy
```

Comment:
I don't see the policy committed here. Where does it come from?

Reply:
we are using dedicated dev and managed workspace for this test, 
as when designed the test  we were told that the environment  will be configured with secrets service account policy but the ReleasePlan , ReleasePlanAdmission and ReleaseStrategy we should create in our test .

Reply:
it would be nice to say that somewhere in the readme then